src/logos/applepay/logo.jsx

```diff
@@ -47,7 +47,30 @@ export const getApplepaySVG = ({ primary }: LogoColors): ElementNode => {
   );
 };
 
-export function ApplePayLogo({
+export function ApplePayLogoExternalImage({
+  logoColor = LOGO_COLOR.DEFAULT,
+  ...props
+}: {
+  logoColor?: $Values<typeof LOGO_COLOR>,
+}): ComponentNode<SVGLogoProps> {
+  const cdnUrl = getLogoCDNUrl(LOGO.APPLEPAY, APPLEPAY_LOGO_COLORS, logoColor);
+
+  return (
+    <SVGLogo
+      {...props}
+      name={LOGO.APPLEPAY}
+      logoColor={logoColor}
+      alt="Apple Pay"
+      aria-label="Apple Pay"
+      cdnUrl={cdnUrl}
+      render={() => {
```

Comment:
Do we still need the `render()` function for these ExternalImage components?

Reply:
We shouldn't.  I'll see about getting it removed!